Fastmail.xcodeproj/project.pbxproj

```diff
@@ -466,7 +506,7 @@
 /* End XCBuildConfiguration section */
 
 /* Begin XCConfigurationList section */
-		C6FF2578241A866F00C896DA /* Build configuration list for PBXProject "DefaultApp" */ = {
```

Comment:
I just noticed some of these changes to project.pbxproj. Not sure what they all mean. Maybe a newer version of Xcode? I could try to back some of them out, or investigate what exactly they mean. E.g., I don't RECALL renaming the Build Configuration list.